### ISSUE TITLE
Insert ‘of’

### ISSUE THREAD

**Issue by woodard** [ccoutant/dwarf-locations#49]
https://github.com/ccoutant/dwarf-locations/blob/2e9102f1248c3247be5611725216ddc90f0d8c32/003-clarifications-loc.txt#L192

[non-normative] The undefined location storage represents a piece or all of an object

For grammatical consistency I think it should be “a piece of or all of an object”

**Comment by t-tye** [comment]
I agree. Just for reference this text appears in a few places in the DWARF 5 spec:

2.6.1.1.1 Empty Location Descriptions
An empty location description consists of a DWARF expression containing no
operations. It represents a piece or all of an object that is present in the source but
not in the object code (perhaps due to optimization).

2.6.1.1.3 Register Location Descriptions
A register location description consists of a register name operation, which
represents a piece or all of an object located in a given register.

2.6.1.1.4 Implicit Location Descriptions
An implicit location description represents a piece or all of an object which has
no actual location but whose contents are nonetheless either known or known to
be undefined.
